src/databricks/labs/ucx/hive_metastore/tables.scala

```diff
@@ -79,7 +79,7 @@ def getInventoryDatabase(): String={
 }
 
 val inventoryDatabase = getInventoryDatabase()
-var df = metadataForAllTables(spark.sharedState.externalCatalog.listDatabases(), failures)
+var df = metadataForAllTables(spark.sharedState.externalCatalog.listDatabases().filter(_ != s"$inventoryDatabase"), failures)
```

Comment:
```suggestion
var df = metadataForAllTables(spark.sharedState.externalCatalog.listDatabases().filter(_ != inventoryDatabase), failures)
```
check if this works

Reply:
Tested the same and the table was not having the inventory database.Working as expected

Reply:
why do you need to interpolate a string within another string? `inventoryDatabase` is a string already.